lib/service/checkpayments.go

```diff
@@ -62,18 +63,24 @@ func (svc *LndhubService) TrackOutgoingPaymentstatus(ctx context.Context, invoic
 			return
 		}
 		if payment.Status == lnrpc.Payment_FAILED {
-			svc.Logger.Infof("Failed payment detected: %v", payment)
+			svc.Logger.Infof("Failed payment detected: hash %s, reason %s", payment.PaymentHash, payment.FailureReason)
 			//todo handle failed payment
-			//return svc.HandleFailedPayment(ctx, invoice, entry, fmt.Errorf(payment.FailureReason.String()))
+			err = svc.HandleFailedPayment(ctx, invoice, entry, fmt.Errorf(payment.FailureReason.String()))
+			if err != nil {
+				svc.Logger.Errorf("Error handling failed payment %v: %s", invoice, err.Error())
```

Comment:
this sounds like an error that needs attention? maybe this should be a sentry notification?